code/modules/research/xenoarchaeology/finds/finds_special.dm

```diff
@@ -150,6 +154,30 @@
 		B.blood_DNA[M.dna.unique_enzymes] = M.dna.b_type
 		M.vessel.remove_reagent(/singleton/reagent/blood,rand(25,50))
 
+/obj/item/vampiric/proc/bloodcall_final(var/mob/living/carbon/human/M)
+	if(istype(M))
+		var/target = pick(M.organs_by_name)
```

Comment:
Should exclude the head, as this is instant death. May also wish to exclude the lower torso as this is effectively crippling.

Reply:
I've altered the consequences completely. There is no opportunity for instant death now.